scripts/regression.py

```diff
@@ -119,7 +119,7 @@ def generate_raw_table(test_dict):
             meminfo = commands.getoutput("cat %s/meminfo" % sysinfodir)
             lspci = commands.getoutput("cat %s/lspci_-vvnn" % sysinfodir)
             partitions = commands.getoutput("cat %s/partitions" % sysinfodir)
-            fdisk = commands.getoutput("cat %s/fdisk_-l" % sysinfodir)
+            fdisk = commands.getoutput("cat %s/fdisk_-line" % sysinfodir)
```

Comment:
Good catch, sorry about that (I always checked what it touches, but I must have skipped this one...)